rmgpy/rmg/model.py

```diff
@@ -1414,11 +1416,18 @@ def addSeedMechanismToCore(self, seedMechanism, react=False):
         seedMechanism = database.kinetics.libraries[seedMechanism]
         
         rxns = seedMechanism.getLibraryReactions()
+        
         for rxn in rxns:
             if isinstance(rxn,LibraryReaction) and not (rxn.library in libraryNames): #if one of the reactions in the library is from another library load that library
                 database.kinetics.libraryOrder.append((rxn.library,'Internal'))
                 database.kinetics.loadLibraries(path=path,libraries=[rxn.library])
                 libraryNames = database.kinetics.libraries.keys()
+            if isinstance(rxn,TemplateReaction) and not (rxn.family in familyNames):
+                logging.warning('loading reaction {0} originally from family {1} as a library reaction'.format(str(rxn),rxn.family))
+                rxn = LibraryReaction(reactants=rxn.reactants[:], products=rxn.products[:],\
+                 library=seedMechanism.name, specificCollider=rxn.specificCollider, kinetics=rxn.kinetics, duplicate=rxn.duplicate,\
```

Comment:
The forward slashes are unnecessary here and in the same place below.

Reply:
fixed